### PR TITLE
xy_13

### PR DESCRIPTION
**Pull request by hCraker**
Closes: #248 
Original NCL Plot:
image

Python equivalent:
image

### CODE REVIEW

File: Plots/XY/NCL_xy_13.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
+"""
+NCL_xy_12.py
+===============
+This script illustrates the following concepts:
+   - Adding error bars on an XY plot
```
Comment (michaelavs) [inline Plots/XY/NCL_xy_13.py:5]: I think this needs to be indented one space

File: Plots/XY/NCL_xy_13.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
+"""
+NCL_xy_12.py
```
Comment (michaelavs) [inline Plots/XY/NCL_xy_13.py:2]:
```suggestion
NCL_xy_13.py
```

File: Plots/XY/NCL_xy_13.py (hunk 1 above)
Comment (hCraker) [reply Plots/XY/NCL_xy_13.py]: Good catch! I just fixed it

**Review by michaelavs — APPROVED** [review]

**Comment by hCraker** [comment]
@erogluorhan @michaelavs Given the new workflow, should I tag this as ready to merge now or do I still need to request review from @jukent?

**Comment by erogluorhan** [comment]
> @erogluorhan @michaelavs Given the new workflow, should I tag this as ready to merge now or do I still need to request review from @jukent?

In general, we should be good with 2 approving reviews without having several people in the loop from now on. However, my concern in particular with this PR is that although the PR still did not pass build tests (i.e. it is not all green, which means we cannot guarantee if it would not break the `master` branch when merged), we have two approving reviews. In other words, as a rule of thumb, please make sure to see PRs all green with their build tests in order to approve them.

**Comment by hCraker** [comment]
> In general, we should be good with 2 approving reviews without having several people in the loop from now on. However, my concern in particular with this PR is that although the PR still did not pass build tests (i.e. it is not all green, which means we cannot guarantee if it would not break the `master` branch when merged), we have two approving reviews. In other words, as a rule of thumb, please make sure to see PRs all green with their build tests in order to approve them.

Okay thank you for clarifying. Have we made any progress on figuring out why the build tests are stalling?

**Pull request merged**